Makefile

```diff
@@ -0,0 +1,46 @@
+artifact_name       := company-accounts.web.ch.gov.uk
+commit              := $(shell git rev-parse --short HEAD)
+tag                 := $(shell git tag -l 'v*-rc*' --points-at HEAD)
+version             := $(shell if [[ -n "$(tag)" ]]; then echo $(tag) | sed 's/^v//'; else echo $(commit); fi)
+artifactory_publish := $(shell if [[ -n "$(tag)" ]]; then echo release; else echo dev; fi)
```

Comment:
`artifactory_publish` isn't used anywhere? If you haven't made a `publish` target for a reason, then remove this line out.